File: Лабораторна робота 9/Lab_9/lens.py
class Lens:
    def __init__(self, focal_length):
        """
        Creates a new lens with the given focal length.
        """
        self.focal_length = focal_length

    def zoom(self, zoom_factor):
        """
        Zooms the lens by the given factor.
        """
        self.focal_length *= zoom_factor
    
    def focus(self, distance):
        """
        Focuses on an object at the given distance.
        """
        print("Фокусування на об'єкті на відстані {} метрів.".format(distance))

    def aperture(self, aperture):
        """
        Sets the aperture to the given value.
        """
        print("Aperture is {}".format(aperture))

    def __str__(self):
        return "Lens with focal length {}".format(self.focal_length)

File: Лабораторна робота 2/Lab_2/Lab_2_2/src/KI306/Kyianets/Lab2/Lens.java
package KI306.Kyianets.Lab2;

public class Lens {
    private String focalLength;
    private String aperture;

    // Конструктор класу Lens
    public Lens(String focalLength, String aperture) {
        this.focalLength = focalLength;
        this.aperture = aperture;
    }

    // Геттер для отримання фокусної відстані
    public String getFocalLength() {
        return focalLength;
    }

    // Додайте інші методи та конструктори за необхідності.
}
